website/server/controllers/api-v3/members.js

```diff
@@ -408,8 +416,8 @@ api.transferGems = {
     if (receiver.preferences.pushNotifications.giftedGems !== false) {
       sendPushNotification(receiver,
         {
-          title: res.t('giftedGems'),
-          message: res.t('giftedGemsInfo', {amount: gemAmount, name: byUsername}),
+          title: res.t('giftedGems', langs[0]),
```

Comment:
res.t automatically passes the user language to the translation function. If you want to pass a different value you'll want to use `i18n.t`